tests/functional/commands/test_status_command.py

```diff
@@ -93,6 +93,35 @@ def test_experiment_wout_success_wout_ac(clean_db, single_without_success, capsy
     assert captured == expected
 
 
+def test_experiment_number_same_list_status(clean_db, only_invalidalgo_experiments_db,
+                                            single_without_success, capsys):
+    """Test status and list command output the consistent number of experiments"""
+    orion.core.cli.main(['status'])
+
+    captured = capsys.readouterr().out
+
+    expected = """\
+test_single_exp-v1
```

Comment:
Shouldn't the experiment with invalid algo be listed here as well? There is no `supernaedo2-dendi`. :thinking:

Reply:
Since root_id and db _id is not the same for `supernaedo2-dendi`, so it is ignored. As mentioned above, once we get th invalid algo issue fixed, we need another case to cover this.

Reply:
Oh, I didn't realize this. You should use `ensure_deterministic_id` to make sure it is valid.

I suggest you remove the invalid algo stuff and stash it for next PR. This specific test should not depend on it.

Reply:
okay, let's add this later. I actually should not name it as invalid algorithm but just invalid experiment root_id which does not exist.

Reply:
remove the invalid algo stuff and add it back when we get the safe experiment build work done.